kernel_gateway/gatewayapp.py

```diff
@@ -221,6 +211,17 @@ def _api_changed(self, name, old, new):
     def allow_notebook_download_default(self):
         return os.getenv(self.allow_notebook_download_env, 'False') == 'True'
 
+    def _load_api_module(self, module_name):
+        '''Tries to import the given module name'''
+        api_module = None
+        # some compatibility allowances
+        if module_name == 'jupyter-websocket':
+            module_name = 'kernel_gateway.jupyter_websocket'
+        elif module_name == 'notebook-http':
+            module_name = 'kernel_gateway.notebook_http'
+        api_module = importlib.import_module(module_name)
```

Comment:
Instead of hardcoding these, how about:

```
try:
  api_module = importlib.import_module(module_name)
except ImportError:
  api_module = importlib.import_module('kernel_gateway.'+module_name)
```

It's a cost at startup only, so the initial miss for the two built-in cases isn't a huge deal.